<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>24h du Fourmil1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="horizontalWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Vitesse d'exécution</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="exeSpeedSlider">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBothSides</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addFoodBtn">
           <property name="text">
            <string>Ajouter Nourriture</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addAntHillBtn">
           <property name="text">
            <string>Ajouter fourmilière</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="playStop">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Pause</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="SimulationView" name="graphicsView"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fic&amp;hier</string>
    </property>
    <addaction name="nouvelleSimuBtn"/>
    <addaction name="ouvrirFichierBtn"/>
    <addaction name="separator"/>
    <addaction name="sauvegardeFichierBtn"/>
    <addaction name="sauvegardeSousFichierBtn"/>
    <addaction name="separator"/>
    <addaction name="quitterSimuBtn"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Si&amp;mulation</string>
    </property>
    <addaction name="addAntHillMenuBtn"/>
    <addaction name="addFoodMenuBtn"/>
    <addaction name="separator"/>
    <addaction name="ouvrirParamBtn"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>A&amp;ide</string>
    </property>
    <addaction name="actionManuel"/>
    <addaction name="aProposBtn"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="nouvelleSimuBtn">
   <property name="text">
    <string>&amp;Nouvelle Simulation</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Créer une nouvelle simulation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>Démarrer une nouvelle simulation</string>
   </property>
   <property name="whatsThis">
    <string>Démarrer une nouvelle simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="ouvrirFichierBtn">
   <property name="text">
    <string>&amp;Ouvrir</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ouvrir une simulation existante&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="sauvegardeFichierBtn">
   <property name="text">
    <string>&amp;Enregister</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sauvegarde votre simulation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="quitterSimuBtn">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="toolTip">
    <string>Quitter le programme</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="realTInfoBtn">
   <property name="text">
    <string>&amp;Infos en temps réel</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Affiche les chiffres de la simulation en temps réel: nombre de colonies, de nourriture...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action0_25">
   <property name="text">
    <string>&amp;0.25</string>
   </property>
  </action>
  <action name="action0_5">
   <property name="text">
    <string>0.&amp;5</string>
   </property>
  </action>
  <action name="action0_75">
   <property name="text">
    <string>0.&amp;75</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>&amp;1</string>
   </property>
  </action>
  <action name="action1_25">
   <property name="text">
    <string>1.25</string>
   </property>
  </action>
  <action name="action1_5">
   <property name="text">
    <string>1.5</string>
   </property>
  </action>
  <action name="action1_75">
   <property name="text">
    <string>1.75</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>&amp;2</string>
   </property>
  </action>
  <action name="addAntHillMenuBtn">
   <property name="text">
    <string>&amp;Nouvelle colonie</string>
   </property>
  </action>
  <action name="addFoodMenuBtn">
   <property name="text">
    <string>&amp;Déposer nourriture</string>
   </property>
  </action>
  <action name="action0_26">
   <property name="text">
    <string>&amp;0.25</string>
   </property>
  </action>
  <action name="action0_6">
   <property name="text">
    <string>0.&amp;5</string>
   </property>
  </action>
  <action name="action0_76">
   <property name="text">
    <string>0.&amp;75</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>&amp;1</string>
   </property>
  </action>
  <action name="action1_26">
   <property name="text">
    <string>1.25</string>
   </property>
  </action>
  <action name="action1_6">
   <property name="text">
    <string>1.5</string>
   </property>
  </action>
  <action name="action1_76">
   <property name="text">
    <string>1.75</string>
   </property>
  </action>
  <action name="action2_2">
   <property name="text">
    <string>&amp;2</string>
   </property>
  </action>
  <action name="ouvrirParamBtn">
   <property name="text">
    <string>&amp;Paramètres</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionManuel">
   <property name="text">
    <string>&amp;Voir le Manuel</string>
   </property>
   <property name="toolTip">
    <string>Ouvre le manuel utilisateur</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="aProposBtn">
   <property name="text">
    <string>&amp;À propos</string>
   </property>
  </action>
  <action name="sauvegardeSousFichierBtn">
   <property name="text">
    <string>Enregister &amp;sous</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SimulationView</class>
   <extends>QGraphicsView</extends>
   <header>simulationview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
